apps/website/src/components/modules/UserOnboardingProcess/UserOnboarding/UserOnboarding.js

```diff
@@ -1,10 +1,10 @@
 import React, { useState } from 'react'
-import FormField from '../../../../../../packages/UI/src/components/organisms/FormField'
-import OpenResponse from '../../../../../../packages/UI/src/components/organisms/OpenResponse'
+import FormField from '../../../../../../../packages/UI/src/components/organisms/FormField'
+import OpenResponse from '../../../../../../../packages/UI/src/components/organisms/OpenResponse'
 import { Checkbox } from '@devlaunchers/components/components/atoms'
 import { Link } from '@devlaunchers/components/components/atoms'
 import OnboardingForm from './OnboardingForm'
-import Button from '../../../../../../packages/UI/src/components/atoms/Button'
+import Button from '../../../../../../../packages/UI/src/components/atoms/Button'
```

Comment:
use `@develaunchers/components/components/atoms/Button`path instead of '../../../../'

Reply:
I am getting this error
![2023-04-02](https://user-images.githubusercontent.com/110208240/229342582-4bd3e26b-c84e-4891-8532-55db9d7e12ce.png)
This is when I am using @develaunchers/components/components/atoms/Button

Reply:
I have try with @develaunchers/components/components/atoms/Button and @develaunchers/components/atoms/Button